### DISCUSSION TITLE
Mattermost Radio

### DISCUSSION THREAD

**Discussion by koox00** [mattermost/community#15] [Fun Ideas]
It would be crazy cool to have a radio inside Mattermost, either music or staff rants about stuff :p
I'm mostly running out of music to listen to so somebody else picking the music would feel great :stuck_out_tongue:

**Comment by gigawhitlocks** [comment]
A synced stream to participants in a channel could be really cool

Could be cool w/ video too (like a private Twitch)

**Comment by enahum** [comment]
This would be soooo cool, different shows with different hosts

**Comment by Willyfrog** [comment]
is this something we can bootstrap using a third party app? having a userbase would most likely help other people how awesome this would be.

**Reply by koox00** [reply]
I was thinking more of an Icecast server or equivalent for streaming,
listening though could have no restriction on the stream,
 I mean you could add as many stream URLs you'd like and just listen to through Mattermost Radio plugin :O
